include/npc/npc.h

```diff
@@ -182,6 +395,30 @@ bool item_in_npc_inventory(npc_t *npc, char *item_id);
 
 // "GET" FUNCTIONS ------------------------------------------------------------
 
+/*
+ * Gets a quest with a specific id
+ * 
+ * Parameters:
+ * - npc: the npc
+ * - id: the quest's id
+ * 
+ * Returns:
+ * - the quest if present, else NULL
+*/
+npc_quest_t *get_npc_quest(npc_t *npc, char *id);
```

Comment:
maybe a slight nit-pick but because there are so many different types of `id`s I think it's be best if they were specified in the function declarations
```suggestion
npc_quest_t *get_npc_quest(npc_t *npc, char *quest_id);
```

Reply:
Yeah, I agree. That would improve readability/reduce confusion a lot.